tests/e2e/utils/mock-requests.js

```diff
@@ -844,4 +844,10 @@ export default class MockRequests {
 			methods
 		);
 	}
+
+	async fulfillUsersPreferences( payload = {} ) {
```

Comment:
@ankitrox What does this line do exactly please?

Reply:
@asvinb It will prevent the preferences to be stored in DB so that next time the tests run, those should not fail.